### ISSUE TITLE
What happens in BACK-END? 

### ISSUE THREAD

**Issue by hamitbugrabayram** [UZ-SLAMLab/ORB_SLAM3#921]
Hi, I am new on Visual Inertial SLAM (actually I'm a student and it's my term project). I try to understand back back-end optimization process. So, I have some questions about that. 

Firstly, how to calculate the scale factor? In the paper, when the scale converges to 1, the mono inertial acts like stereo inertial mode. How it's possible? 

Secondly, how to use IMU measurements in the initialization process and all processes? Why do we preintegrate IMU? 

In summary, what is the mathematical expression of that how fix the scale problem only monocular slam using with IMU in algorithm. Please help me! I'm about to go crazy!!!

**Comment by hamitbugrabayram** [comment]
+++
